kiota_abstractions/request_information.py

```diff
@@ -45,7 +45,7 @@ def __init__(self) -> None:
         self.query_parameters: Dict[str, QueryParams] = {}
 
         # The Request Headers
-        self.headers: Dict[str, str] = {}
+        self.headers: Dict[str, List[str]] = {}
```

Comment:
Can't we just use a set instead of a list from this point, so we don't worry about duplicate values?
```suggestion
        self.headers: Dict[str, Set[str]] = {}
```

Reply:
See https://github.com/microsoft/kiota-abstractions-python/pull/39/commits/4d22efd4a9bdeae2be2067e9deb20e873d4631bb